Document button cursor change (#2023)
tailwindlabs/tailwindcss.com: src/docs/upgrade-guide.mdx

}
```

#### Button cursor change

> In v3, `button` elements would have a `cursor: pointer` by default. This was changed in v4 to be closer to browser defaults.

Reviewer: ```suggestion
In v3, `button` elements had `cursor: pointer` by default. This was changed in v4 to be closer to browser defaults.
```
